### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by maciejmrowiec**

### CODE REVIEW

**Comment by maciejmrowiec** [comment]
@nickanderson 
Should we do those upgrades?

**Comment by nickanderson** [comment]
Do what upgrades?

File: .travis.yml [hunk 1]
```diff
@@ -9,3 +10,4 @@ install:
 
 script:
 - /var/cfengine/bin/cf-promises -f $TRAVIS_BUILD_DIR/promises.cf
+- /var/cfengine/bin/cf-promises -f $TRAVIS_BUILD_DIR/update.cf
```
Comment (nickanderson) [inline .travis.yml]: YES

Comment (nickanderson) [inline .travis.yml]: We should test update.cf in addtion to promises.cf.

**Pull request merged**